### PR TITLE
Tracability

### PR DESCRIPTION
**Pull request by psibomana**

### CODE REVIEW

File: apps/templates/farmer/edit-farm.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            <!-- /.row -->
+        </div><!-- /.container-fluid -->
+    </section>
+    <!-- /.content -->
+</div>
+
+{% endblock content %}
+
+{% block js_editable %}
+<script type="text/javascript" lang="javascript">
```
Comment (psibomana) [inline apps/templates/farmer/edit-farm.html:113]: @Check-karim , please use flask forms instead
https://python-adv-web-apps.readthedocs.io/en/latest/flask_forms.html

https://www.digitalocean.com/community/tutorials/how-to-use-web-forms-in-a-flask-application

File: apps/templates/farmer/edit-farmer.html [hunk 2]
```diff
@@ -134,6 +139,59 @@ <h3 class="profile-username text-center">
 
 {% endblock content %}
 
+{% block js_editable %}
+<script type="text/javascript" lang="javascript">
```
Comment (psibomana) [inline apps/templates/farmer/edit-farmer.html:143]: Same here @Check-karim , use Flask Forms.
https://www.digitalocean.com/community/tutorials/how-to-use-web-forms-in-a-flask-application

https://python-adv-web-apps.readthedocs.io/en/latest/flask_forms.html

File: index
Comment (psibomana) [inline index:1]: Do we need to have this file added to the code base?

File: records
Comment (psibomana) [inline records:1]: Do we need to have this file added to the code base?

**Review by psibomana — COMMENTED** [review]
Note the changes done in the models, with the renaming of the fields.

I have sent a db dump with the new data structure on email.

File: apps/templates/farmer/edit-farm.html (hunk 1 above)
Comment (Check-karim) [reply apps/templates/farmer/edit-farm.html]: @psibomana done

File: apps/templates/farmer/edit-farmer.html (hunk 2 above)
Comment (Check-karim) [reply apps/templates/farmer/edit-farmer.html]: @psibomana done

File: index
Comment (Check-karim) [reply index:1]: file removed it was unneeded

File: records
Comment (Check-karim) [reply records:1]: file removed it was unneeded

**Review by psibomana — COMMENTED** [review]
@Check-karim, I am merging this one and created a new upload branch to fix the upload alone. 
Nshaka babe baragira tests yivyo dufise deja

**Pull request merged**